### ISSUE TITLE
Problem when executing step 6: call install-windows.bat

### ISSUE THREAD

**Issue by freemank1224** [fkeidel/CarND-Term2-ide-profile-VisualStudio#1]
I use Win10 pro 64bit and VS2017 community edition. The setting line was not modified , current setting was shown below: 
set "CarNDEKFProjectPlatform=x86"
set "CarNDEKFProjectToolset=v140"
set "CarNDEKFProjectBuildType=Debug"

When I run the install-windows.bat, 

> The following packages will be built and installed:
>   * libuv[core]:x86-windows
>   * openssl[core]:x86-windows
>     uwebsockets[core]:x86-windows
>   * zlib[core]:x86-windows
> Additional packages (*) will be modified to complete this operation.
> Starting package 1/4: zlib:x86-windows
> Building package zlib:x86-windows...
> Warning: The following VS instances are exluded because the English language pack is unavailable.
>     C:\Program Files (x86)\Microsoft Visual Studio\2017\Community
> Please install the English language pack.
> Could not locate a complete toolset.
> The following paths were examined:
>     C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat
>     C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.11.25503\bin\HostX86\x86\dumpbin.exe
> 已复制         1 个文件。
> 
> Bootstrapping successful for triplet: x86-windows
> CMAKE_PREFIX_PATH=c:\vcpkg\installed\x86-windows;
> 
> 
> CarNDEKFProjectBuildDir=c:\Users\Dyson\OneDrive\WorkDock\Udacity_SDC\Term2_proj\CarND-Extended-Kalman-Filter-Project\products\cmake.msbuild.windows.x86.v140
> cmake.exe -G "Visual Studio 14 2015" "c:\Users\Dyson\OneDrive\WorkDock\Udacity_SDC\Term2_proj\CarND-Extended-Kalman-Filter-Project\ide_profiles\VisualStudio\\..\.."
> 
> CMake Error at CMakeLists.txt:1 (project):
>   Failed to run MSBuild command:
> 
>     MSBuild.exe
> 
>   to get the value of VCTargetsPath:
> 
>     系统找不到指定的文件。
> 
> 
> 
> -- Configuring incomplete, errors occurred!

Is this all because I don't have a English language package? Please help me on this.

**Comment by ras0219-msft** [comment]
Yes, unfortunately installing the VS English language pack is needed for some libraries to build inside Vcpkg. Could you try installing it? It's very easy -- just select English in the "Language Packs" tab in the VS2017 installer.
